acms-react-app/src/Pages/Admin_Pages/AppointmentNew_Page/Calendar.js

```diff
@@ -20,13 +21,66 @@ export default function Calendar() {
     const [selectedTime, setSelectedTime] = useState('10:00');
     const [selectedService, setSelectedService] = useState(null);
     const [comments, setComments] = useState('');
+    const navigate = useNavigate();
+
+    const [customerId, setCustomerId] = useState('');
+    const [vehicleId, setVehicleId] = useState('');
+
+    const handleSubmit = () => {
+
+        const appointmentData = {
+            customerId,
+            vehicleId,
+            appointmentDate: selectDate.format("YYYY-MM-DD") + 'T' + selectedTime,
+            services: selectedService,
+            comments,
+            status: 'PENDING'
+        };
+        if (!customerId || !vehicleId) {
+            console.error("Customer ID and Vehicle ID are required.");
+            console.log("Appointment data:", appointmentData)
+            return;
+        }
+        console.log("Submitting appointment:", appointmentData);
+
+        const url = `http://localhost:8080/api/v1/customers/${customerId}/appointments`;
+
+        fetch(url, {
```

Comment:
We have been using axios, make sure to keep it that way, I am currently adding services so we won't have all these requests in the middle of every file.